python/python/bystro/supervised_ppca/gf_generative_pt.py

```
@@ -606,6 +606,71 @@
+        if sherman_woodbury is False:
+            prec = self.get_precision()
+            coefs = np.dot(self.W_, prec)
+        else:
+            M = np.dot(self.W_, self.W_.T) + np.eye(self.n_components) * self.sigma2_
+            coefs = np.dot(self.W_.T, la.inv(M)).T
+
+        return np.dot(X, coefs.T)
+ 
+    def transform_subset(
+        self,
+        X: NDArray[np.float_],
```

Ruff

`np.float_` will be removed in NumPy 2.0. Use `numpy.float64` instead.

Replace with `numpy.float64`

(NPY201)